src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -20,7 +20,7 @@
  */
 typedef struct uct_rc_mlx5_iface_config {
     uct_rc_mlx5_iface_common_config_t super;
-    uct_rc_fc_config_t                fc;
+    uct_rc_common_config_t            rc_common;
```

Comment:
looks like the configuration hierarchy does not fit the iface hierarchy.. we can see how to fix it in another PR
i would expect it to look like this:
```
typedef struct uct_rc_mlx5_iface_config {
    uct_rc_iface_config_t             super;
    uct_rc_mlx5_iface_common_config_t rc_mlx5_common;
}
```